sql/moz-fx-data-marketing-prod/ga_derived/firefox_whatsnew_summary_v1/query.sql

```diff
@@ -0,0 +1,34 @@
+WITH wnp_visits AS (
+  SELECT
+    date,
+    visit_identifier,
+    hit_number,
+    bounces,
+    first_interaction,
+    TRIM(page_path_level1, '/') AS locale,
+    page_level_2 AS version,
+    country,
+    IF(hit_number = first_interaction AND bounces = 1, TRUE, FALSE) AS is_bounce
+  FROM
+    `moz-fx-data-marketing-prod.ga_derived.www_site_hits_v1`
+  WHERE
+    date = @submission_date
+    AND page_name LIKE '/firefox/%/whatsnew%'
+    AND hit_type = 'PAGE'
+    AND page_level_1 = 'firefox'
+    AND page_level_3 = 'whatsnew'
```

Comment:
With the new `page_level_#` conditions the `page_name` condition is no longer necessary, but it does help clarify the intent.  Alternatively, a comment could be used to explicitly explain the intent.
```suggestion
    AND hit_type = 'PAGE'
    -- Match page paths like "/{locale}/firefox/{version}/whatsnew/..."
    AND page_level_1 = 'firefox'
    AND page_level_3 = 'whatsnew'
```